### PR TITLE
Update README badge links

### LINKED ISSUE #1180: readme badges system needs update
* Cookiecutter version: All
* Template project url: n/a
* Python version: n/a
* Operating System: /na

### Description:

After project movement from Audrey's repository to cookiecutter org, we need to update all services that get data from repo to make CI/CD and code analysis. 

Currently already not working: 

- [ ] Code health badge (https://landscape.io)
- [ ] All CI/CD steps, as shown in #1181 
   - [ ] Travis
   - [ ] appveyor/pr 
   - [ ] codecov
   - [ ] readthedocs

### PR DESCRIPTION
**Pull request by luzfcb**
Fix #1180 

All CI/CD steps, as shown in #1181 
   - [X] Travis
   - [X] appveyor/pr 
   - [X] codecov
   - [X] scrutinizer-ci
   - [x] remove landscape.io

About the landscape.io integration: Since the project moved to an organization, landscape.io can't find the project anymore, so I had to [create it again in landscape.io](https://landscape.io/repository/58537/status), however, the webhook integration process is never completed. The issue described here: https://github.com/landscapeio/landscape-issues/issues/310

### CODE REVIEW

**Comment by michaeljoseph** [comment]
Thoughts on dropping landscape for now then?

**Comment by michaeljoseph** [comment]
@luzfcb should try update readthedocs in this change also?

File: README.rst [hunk 1]
```diff
@@ -8,14 +8,14 @@ Cookiecutter
 .. image:: https://img.shields.io/pypi/pyversions/cookiecutter.svg
         :target: https://pypi.python.org/pypi/cookiecutter
 
-.. image:: https://travis-ci.org/audreyr/cookiecutter.svg?branch=master
-        :target: https://travis-ci.org/audreyr/cookiecutter
+.. image:: https://travis-ci.org/cookiecutter/cookiecutter.svg?branch=master
+        :target: https://travis-ci.org/cookiecutter/cookiecutter
```
Comment (michaeljoseph) [inline README.rst]: The builds linked to PRs point to `travis-ci.com` not `.org` 🤔

**Comment by michaeljoseph** [comment]
I just granted RDD and travis cookiecutter org access.

**Comment by hackebrot** [comment]
I haven't used landscape.io. Happy to remove it unless folks find it useful? 🤷‍♂

File: README.rst (hunk 1 above)
Comment (luzfcb) [reply README.rst]: We have travis.org and travis.com enabled, so I got confused. Either way, Travis-ci recommends [1][2] migrating to travis-ci.com, so let's do it.

[1] https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps
[2] https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/

**Comment by luzfcb** [comment]
I really like the landscape.io UI, but I'm fine to let it go, then I removed the landscape.io from the README.

**Pull request merged**